CRM/Xcm/MatchingEngine.php

```diff
@@ -25,6 +25,9 @@ class CRM_Xcm_MatchingEngine {
   /** Configuration for this engine */
   protected $config = NULL;
 
+  /** @var bool  */
+  protected $isStreetAddressParsingEnabled = FALSE;
```

Comment:
Even though it's not the case for the existing code, I think we should use PHP type hints instead of phpdoc type hints.

This property is initialized in the constructor so it probably doesn't need a default value.

Reply:
I have changed this